cmd/svcat/output/class.go

```diff
@@ -32,16 +32,29 @@ func getClassStatusText(status v1beta1.ClusterServiceClassStatus) string {
 
 func writeClassListTable(w io.Writer, classes []v1beta1.ClusterServiceClass) {
```

Comment:
I feel like there's a table writing library we ought to be using.

Reply:
there is and we're using it. It just doesn't support this feature